client/src/pages/Configurations/ConfigBuild/index.tsx

```diff
@@ -733,17 +733,19 @@ export function CustomCodeModal({
   // TODO: this should come from the server.
   // Maybe get this info as part of the seed script?
   const systemValues = [
+    { name: 'Select system', value: '' },
     { name: 'ICD-10', value: 'icd-10' },
     { name: 'SNOMED', value: 'snomed' },
     { name: 'RxNorm', value: 'rxnorm' },
     { name: 'LOINC', value: 'loinc' },
+    { name: 'OTHER', value: 'other' },
```

Comment:
Let's change this so it's consistent with other places we display it.
```suggestion
    { name: 'Other', value: 'other' },
```

For example
<img width="317" height="219" alt="image" src="https://github.com/user-attachments/assets/a173aefb-bbb8-46b3-9947-0e7d7d9ce0af" />